<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>invite_reception</class>
 <widget class="QDialog" name="invite_reception">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1131</width>
    <height>754</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>1351</width>
     <height>871</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1291</width>
      <height>801</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
	
	
	background-image: url(:/img/ressources/2.png);
	

		

font-family:century gothic;
font-size:18px;
}
QTabWidget
{
 background:#333;
}


QFrame
{
 background:#333;
	background-color: rgb(107, 107, 107);
	background-color: rgb(188, 193, 101);
	background-color: rgb(191, 165, 165);
	background-color: rgb(89, 14, 54);
	
}


QPushButton
{
background:red;
	background-color: rgb(255, 107, 114);
border-raduis:60px;
}
QLabel
{
color:white;
}
QPushButton
{
color:white;
border-radius:15px;
}
QPushButton:hover
{
color:#333;
border-raduis:15px;
background:#49ebff;
}
QLineEdit
{
background:transparent;
border:none;
color:#ffffff;
color: rgb(62, 124, 61);
	color: rgb(0, 255, 213);
	
border-button:3ptx solid #ffffff;
	color: rgb(6, 255, 56);
	
}
QComboBox
{
background:transparent;
border:none;
color:#717072;
border-bottom:1px solid #717072 }</string>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="reception">
     <attribute name="title">
      <string>RECEPTION</string>
     </attribute>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>361</width>
        <height>371</height>
       </rect>
      </property>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>831</width>
        <height>631</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="supprimer">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>420</y>
         <width>131</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-width: 1px; border-radius: 4px;
color: rgb(255, 255, 255);
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;

	selection-background-color: rgb(187, 187, 187);
	selection-color: rgb(60, 63, 65);</string>
       </property>
       <property name="text">
        <string>supprimer</string>
       </property>
       <property name="icon">
        <iconset resource="ressources.qrc">
         <normaloff>:/img/ressources/127960661_183172423452732_4500599527248670551_n.png</normaloff>:/img/ressources/127960661_183172423452732_4500599527248670551_n.png</iconset>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_supprimer">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>420</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-width: 1px; border-radius: 4px;
color: rgb(255, 255, 255);
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;

	selection-background-color: rgb(187, 187, 187);
	selection-color: rgb(60, 63, 65);</string>
       </property>
       <property name="placeholderText">
        <string>        ID_RECEP</string>
       </property>
      </widget>
      <widget class="QPushButton" name="imprimer">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>520</y>
         <width>131</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-width: 1px; border-radius: 4px;
color: rgb(255, 255, 255);
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;

	selection-background-color: rgb(187, 187, 187);
	selection-color: rgb(60, 63, 65);</string>
       </property>
       <property name="text">
        <string>imprimer</string>
       </property>
       <property name="icon">
        <iconset resource="ressources.qrc">
         <normaloff>:/img/ressources/128411451_681880159137485_1506548420732995776_n.png</normaloff>:/img/ressources/128411451_681880159137485_1506548420732995776_n.png</iconset>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit1">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>520</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-width: 1px; border-radius: 4px;
color: rgb(255, 255, 255);
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;

	selection-background-color: rgb(187, 187, 187);
	selection-color: rgb(60, 63, 65);</string>
       </property>
       <property name="placeholderText">
        <string>        ID_RECEP</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>130</y>
         <width>121</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>century gothic</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>180</y>
         <width>191</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>century gothic</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Ville </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>230</y>
         <width>211</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>century gothic</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Salle réservée</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>280</y>
         <width>261</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>century gothic</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Nombres des invités</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ajouter">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>360</y>
         <width>131</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-width: 1px; border-radius: 4px;
color: rgb(255, 255, 255);
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;

	selection-background-color: rgb(187, 187, 187);
	selection-color: rgb(60, 63, 65);</string>
       </property>
       <property name="text">
        <string>ajouter</string>
       </property>
       <property name="icon">
        <iconset resource="ressources.qrc">
         <normaloff>:/img/ressources/128362656_213598176780517_3160026623662294284_n.png</normaloff>:/img/ressources/128362656_213598176780517_3160026623662294284_n.png</iconset>
       </property>
      </widget>
      <widget class="QPushButton" name="modifier">
       <property name="geometry">
        <rect>
         <x>660</x>
         <y>360</y>
         <width>131</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-width: 1px; border-radius: 4px;
color: rgb(255, 255, 255);
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;

	selection-background-color: rgb(187, 187, 187);
	selection-color: rgb(60, 63, 65);</string>
       </property>
       <property name="text">
        <string>modifier</string>
       </property>
       <property name="icon">
        <iconset resource="ressources.qrc">
         <normaloff>:/img/ressources/128586817_297996038251568_153686129387149344_n.png</normaloff>:/img/ressources/128586817_297996038251568_153686129387149344_n.png</iconset>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_id_recep">
       <property name="geometry">
        <rect>
         <x>650</x>
         <y>130</y>
         <width>171</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-width: 1px; border-radius: 4px;
color: rgb(255, 255, 255);
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;

	selection-background-color: rgb(187, 187, 187);
	selection-color: rgb(60, 63, 65);</string>
       </property>
       <property name="placeholderText">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_nombre_des_invites">
       <property name="geometry">
        <rect>
         <x>650</x>
         <y>280</y>
         <width>171</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-width: 1px; border-radius: 4px;
color: rgb(255, 255, 255);
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;

	selection-background-color: rgb(187, 187, 187);
	selection-color: rgb(60, 63, 65);</string>
       </property>
       <property name="placeholderText">
        <string>NB invités</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_salle_reservee">
       <property name="geometry">
        <rect>
         <x>650</x>
         <y>220</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-width: 1px; border-radius: 4px;
color: rgb(255, 255, 255);
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;

	selection-background-color: rgb(187, 187, 187);
	selection-color: rgb(60, 63, 65);</string>
       </property>
       <property name="placeholderText">
        <string>Salle</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_ville_recep">
       <property name="geometry">
        <rect>
         <x>650</x>
         <y>180</y>
         <width>171</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-width: 1px; border-radius: 4px;
color: rgb(255, 255, 255);
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;

	selection-background-color: rgb(187, 187, 187);
	selection-color: rgb(60, 63, 65);</string>
       </property>
       <property name="placeholderText">
        <string>Ville</string>
       </property>
      </widget>
     </widget>
     <zorder>frame_2</zorder>
     <zorder>tableView</zorder>
    </widget>
    <widget class="QWidget" name="invite">
     <attribute name="title">
      <string>INVITE</string>
     </attribute>
     <widget class="QTableView" name="tableView_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>381</width>
        <height>371</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_10">
      <property name="geometry">
       <rect>
        <x>910</x>
        <y>300</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Retour</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>811</width>
        <height>631</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>50</y>
         <width>81</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>century gothic</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QPushButton" name="rechercher">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>420</y>
         <width>181</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-width: 1px; border-radius: 4px;
color: rgb(255, 255, 255);
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;

	selection-background-color: rgb(187, 187, 187);
	selection-color: rgb(60, 63, 65);</string>
       </property>
       <property name="text">
        <string>rechercher</string>
       </property>
       <property name="icon">
        <iconset resource="ressources.qrc">
         <normaloff>:/img/ressources/icons8-google-web-recherche-50.png</normaloff>:/img/ressources/icons8-google-web-recherche-50.png</iconset>
       </property>
      </widget>
      <widget class="QPushButton" name="supprimerr">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>540</y>
         <width>181</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-width: 1px; border-radius: 4px;
color: rgb(255, 255, 255);
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;

	selection-background-color: rgb(187, 187, 187);
	selection-color: rgb(60, 63, 65);</string>
       </property>
       <property name="text">
        <string>supprimer</string>
       </property>
       <property name="icon">
        <iconset resource="ressources.qrc">
         <normaloff>:/img/ressources/127960661_183172423452732_4500599527248670551_n.png</normaloff>:/img/ressources/127960661_183172423452732_4500599527248670551_n.png</iconset>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_supprimer_2">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>540</y>
         <width>171</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-width: 1px; border-radius: 4px;
color: rgb(255, 255, 255);
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;

	selection-background-color: rgb(187, 187, 187);
	selection-color: rgb(60, 63, 65);</string>
       </property>
       <property name="placeholderText">
        <string>         ID_INV</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_rechercher_invite">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>420</y>
         <width>171</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-width: 1px; border-radius: 4px;
color: rgb(255, 255, 255);
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;

	selection-background-color: rgb(187, 187, 187);
	selection-color: rgb(60, 63, 65);</string>
       </property>
       <property name="placeholderText">
        <string>          ID_INV</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>100</y>
         <width>351</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>century gothic</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Nombre de place</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>160</y>
         <width>181</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>century gothic</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Numero de table</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>220</y>
         <width>181</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>century gothic</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Email de l'invité</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_numero_table">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>160</y>
         <width>231</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-width: 1px; border-radius: 4px;
color: rgb(255, 255, 255);
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;

	selection-background-color: rgb(187, 187, 187);
	selection-color: rgb(60, 63, 65);</string>
       </property>
       <property name="placeholderText">
        <string>num table</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_id_inv">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>50</y>
         <width>231</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-width: 1px; border-radius: 4px;
color: rgb(255, 255, 255);
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;

	selection-background-color: rgb(187, 187, 187);
	selection-color: rgb(60, 63, 65);</string>
       </property>
       <property name="placeholderText">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_mail_invite">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>220</y>
         <width>231</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-width: 1px; border-radius: 4px;
color: rgb(255, 255, 255);
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;

	selection-background-color: rgb(187, 187, 187);
	selection-color: rgb(60, 63, 65);</string>
       </property>
       <property name="placeholderText">
        <string>...@gmail.com</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_nomre_de_places_reserves">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>100</y>
         <width>231</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-width: 1px; border-radius: 4px;
color: rgb(255, 255, 255);
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;

	selection-background-color: rgb(187, 187, 187);
	selection-color: rgb(60, 63, 65);</string>
       </property>
       <property name="placeholderText">
        <string>NB places</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ajouterr">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>290</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-width: 1px; border-radius: 4px;
color: rgb(255, 255, 255);
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;

	selection-background-color: rgb(187, 187, 187);
	selection-color: rgb(60, 63, 65);</string>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
       <property name="icon">
        <iconset resource="ressources.qrc">
         <normaloff>:/img/ressources/128362656_213598176780517_3160026623662294284_n.png</normaloff>:/img/ressources/128362656_213598176780517_3160026623662294284_n.png</iconset>
       </property>
      </widget>
      <widget class="QPushButton" name="mailing">
       <property name="geometry">
        <rect>
         <x>640</x>
         <y>530</y>
         <width>150</width>
         <height>80</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-width: 1px; border-radius: 4px;
color: rgb(255, 255, 255);
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;

	selection-background-color: rgb(187, 187, 187);
	selection-color: rgb(60, 63, 65);</string>
       </property>
       <property name="text">
        <string>mailing</string>
       </property>
      </widget>
      <widget class="QPushButton" name="modifierr">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>290</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-width: 1px; border-radius: 4px;
color: rgb(255, 255, 255);
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;

	selection-background-color: rgb(187, 187, 187);
	selection-color: rgb(60, 63, 65);</string>
       </property>
       <property name="text">
        <string>modifier</string>
       </property>
       <property name="icon">
        <iconset resource="ressources.qrc">
         <normaloff>:/img/ressources/128586817_297996038251568_153686129387149344_n.png</normaloff>:/img/ressources/128586817_297996038251568_153686129387149344_n.png</iconset>
       </property>
      </widget>
     </widget>
     <zorder>frame</zorder>
     <zorder>tableView_2</zorder>
     <zorder>pushButton_10</zorder>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
